<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlay2Widget</class>
 <widget class="QWidget" name="XPlay2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPlay2Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="XVideoWidget" name="VideoWidget">
       <property name="minimumSize">
        <size>
         <width>1280</width>
         <height>760</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XQSliderWidget" name="PlayPos">
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <item>
        <widget class="QPushButton" name="OpenFile">
         <property name="text">
          <string>Open Media File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OpenURL">
         <property name="text">
          <string>OpenURL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="isPlay">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="VolumeSlider">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">XVideoWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>XQSliderWidget</class>
   <extends>QSlider</extends>
   <header location="global">XQSliderWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>OpenFile</sender>
   <signal>clicked()</signal>
   <receiver>XPlay2Widget</receiver>
   <slot>OpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>843</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>808</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isPlay</sender>
   <signal>clicked()</signal>
   <receiver>XPlay2Widget</receiver>
   <slot>PlayOrPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1171</x>
     <y>843</y>
    </hint>
    <hint type="destinationlabel">
     <x>1054</x>
     <y>847</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlayPos</sender>
   <signal>sliderReleased()</signal>
   <receiver>XPlay2Widget</receiver>
   <slot>SliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1157</x>
     <y>797</y>
    </hint>
    <hint type="destinationlabel">
     <x>1286</x>
     <y>714</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlayPos</sender>
   <signal>sliderPressed()</signal>
   <receiver>XPlay2Widget</receiver>
   <slot>SliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1046</x>
     <y>797</y>
    </hint>
    <hint type="destinationlabel">
     <x>1286</x>
     <y>703</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenURL</sender>
   <signal>clicked()</signal>
   <receiver>XPlay2Widget</receiver>
   <slot>OpenURL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>832</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>847</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VolumeSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>XPlay2Widget</receiver>
   <slot>VolumeReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1212</x>
     <y>830</y>
    </hint>
    <hint type="destinationlabel">
     <x>1228</x>
     <y>856</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenFile()</slot>
  <slot>PlayOrPause()</slot>
  <slot>SliderPressed()</slot>
  <slot>SliderReleased()</slot>
  <slot>OpenURL()</slot>
  <slot>VolumeReleased()</slot>
 </slots>
</ui>
